src/neo/Wallets/NEP6/NEP6Account.cs

```diff
@@ -83,5 +83,30 @@ public bool VerifyPassword(string password)
                 return false;
             }
         }
+
+        public bool ChangePassword(string password_old, string password_new)
+        {
+            if (WatchOnly) return true;
```

Comment:
Seems that this is redundant, we should use only one condition for check if it's WatchOnly

Reply:
I think this might be safer, as nep2key == null or key == null might not indicate the account is WatchOnly?